aten/src/ATen/native/native_functions.yaml

```diff
@@ -6999,6 +7002,7 @@
   structured_delegate: addmm.out
   variants: function, method
   dispatch:
+    MkldnnCPU: mkldnn_addmm_cpu
```

Comment:
Since we only need to handle things at the "linear" level, can we avoid changing "addmm"?

Reply:
Unfortunately we have to change addmm since we need it to handle mkldnn layout weights now

Reply:
See my comment in another hunk.